backend/debug_transaction_descriptions.py

```
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+"""
+Debug script to analyze transaction descriptions and find potential Sainsburys matches.
+This helps understand why the SAINSBURYS pattern didn't match any transactions.
+"""
+
+import os
+import sys
+import logging
+from typing import List, Dict
```

Flake8 (7.2.0)

[error] 'typing.List' imported but unused

(F401)